vigranumpy/src/core/axistags.cxx

```diff
@@ -646,7 +650,10 @@ void defineAxisTags()
              "contains the given axistype.\n")
         .def("compatible", &AxisInfo::compatible,
              "\naxisinfo1.compatible(axisinfo2) yields True when the two axisinfo objects "
-             "have the same keys and types, or either of the two is 'unknown'.\n")
+             "have the same keys and types, or either of the two is 'unknown'.\n"
+             "Note that, as this does not compare axis resolutions or units, \n"
+             "there may still be mismatch or incompatibility in that regard.")
+        //Equality comparisons (note does not compare axis resolutions or units)
         .def(self == self)
```

Comment:
```suggestion
        .def("__eq__", &AxisInfo::operator==, "Equality comparison. Note: does not compare axis resolutions or units.")
```
Doing this for equality is probably good enough, given that there wasn't any mention before.